nix/pkgs/cc-test-reporter/default.nix

```
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchurl }:
+
+let
+  inherit (stdenv) isLinux isDarwin isWindows;
+
+  platform =
+    if isLinux then "linux" else
+    if isDarwin then "darwin" else
+    if isWindows then "windows" else
+    throw "Unsupported platform: ${stdenv.hostPlatform.system}";
+
+in stdenv.mkDerivation rec {
+  pname = "cc-test-reporter";
+  version = "0.11.1";
+
+  src = fetchurl {
+    url = "https://codeclimate.com/downloads/test-reporter/test-reporter-${version}-${platform}-amd64";
+    hash = lib.getAttr platform {
+      darwin = "sha256-uO9aRL3cJe+KCoC+uN6cBQy8xGQHim6h5Qzw36QO7EY=";
+      linux = "sha256-ne79mW3w9tHJ+3lAWzluuRp6yjWsy4lpdV/KpmjaTa0=";
+      windows = "sha256-8pn8csW9l5xMerZWAwIwWcrO7OLNWEM03yPEMMllaak=";
+    };
+   };
+
+  dontUnpack = true;
+
+  installPhase = ''
+    install -D $src $out/bin/cc-test-reporter
+    chmod +x $out/bin/cc-test-reporter
+  '';
+}
```

```suggestion
    meta = with lib; {
    homepage = "https://codeclimate.com/";
    description = "Command line interface for the Code Climate analysis platform";
    license = licenses.agpl3Only;
    platforms = ["x86_64-linux" "x86_64-darwin" "x86_64-windows"]
  };
}
```
That should prevent it from being buildable on other platforms.